ppr-ui/src/components/mhrTransfers/DeathCertificate.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <div id="death-certificate" class="death-certificate">
+    <v-card flat class="pl-8 rounded" :class="{ 'border-error-left': showFormError }">
```

Comment:
Another minor thing, we only need 1 root element in the template so i'm not sure the div is really required here. You could apply the class and id to the card itself and make it the root element. 
Unless you had a purpose for wrapping it in the div?